app/app/src/main/java/com/example/myapplication/MainActivity.java

```diff
@@ -54,6 +54,48 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
         tv_attention = findViewById(R.id.tv_attention);
 
+        // Pulse animation
+        PulseView pulse = findViewById(R.id.pulse);
+
+        int concentration = 1;
```

Comment:
@Simonster1999 Where in MainActivity can I find something to derive headset concentration level data from? Is it stored as int?